### DISCUSSION TITLE
PyBullet RL Environment Rendering not consistently implemented

### DISCUSSION THREAD

**Discussion by avandekleut** [bulletphysics/bullet3#3741] [General]
Some environments under `pybullet_envs/bullet/` such as `kukaCamGymEnv.py` and `raceCarGymEnv.py` are missing declarations for `self._cam_dist` and other variables needed to call `render(mode='rgb_array)`. In some cases, such as in `cartpole_bullet.py` this variable is declared locally inside of the `render` method. For other environments, they subclass `MJCFBaseBulletEnv` from `pybullet_envs/env_bases.py` which declares `self._cam_dist` and other variables as instance variables of the class.

**Comment by erwincoumans** [comment]
Thanks for the report. There is no activate development on those envs (lack of resources). If you care, you can create a small pull request.
